docs/Dataplex-batchsink.md

```diff
@@ -0,0 +1,183 @@
+and partitioning type is set to Integer.
+* The interval value must be a positive integer.
+
+**Partition Field**: Partitioning column for the BigQuery table. This should be left empty if the
+BigQuery table is an ingestion-time partitioned table.
+
+**Require Partition Filter**: Whether to create a table that requires a partition filter. This value
+is ignored if the table already exists.
+* When this is set to true, table will be created with required partition filter.
+* When this is set to false, table will be created without required partition filter.
+
+**Clustering Order**: List of fields that determines the sort order of the data. Fields must be of type
+INT, LONG, STRING, DATE, TIMESTAMP, BOOLEAN or DECIMAL. Tables cannot be clustered on more than 4 fields.
+This value is only used when the BigQuery table is automatically created and ignored if the table
+already exists.
+
+
+Compatible changes fall under the following categories:
+* the pipeline schema contains nullable fields that do not exist in the BigQuery schema.
+  In this case, the new fields will be added to the BigQuery schema.
+* the pipeline schema contains nullable fields that are non-nullable in the BigQuery schema.
+  In this case, the fields will be modified to become nullable in the BigQuery schema.
+* the pipeline schema does not contain fields that exist in the BigQuery schema.
+  In this case, those fields in the BigQuery schema will be modified to become nullable.
+
+Incompatible schema changes will result in pipeline failure.
+
+**Schema**: Schema of the data to write.
+If a schema is provided, it must be compatible with the table schema in BigQuery.
+
+Data Type Mappings from CDAP to BigQuery Asset
+----------
+The following table lists out different CDAP data types, as well as the
+corresponding BigQuery data type for each CDAP type, for updates and upserts.
+
+| CDAP type      | BigQuery type |
+|----------------|---------------|
+| array          | repeated      |
+| boolean        | bool          |
+| bytes          | bytes         |
+| date           | date          |
+| datetime       | datetime, string|
+| decimal        | numeric       |
+| double / float | float64       |
+| enum           | unsupported   |
+| int / long     | int64         |
+| map            | unsupported   |
+| record         | struct        |
+| string         | string, datetime(Should be ISO 8601 format)|
+| time           | time          |
+| timestamp      | timestamp     |
+| union          | unsupported   |
+
+For inserts, the type conversions are the same as those used in loading Avro
+data to BigQuery; the table is available
+[here](https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-avro#avro_conversions).
+
+Storage Bucket specific fields
+----------
+
+
+**Path Suffix:** Time format for the output directory that will be appended to the path.
```

Comment:
Is this compatible with hive-style partitioning?

Reply:
path suffix is for raw zone. Need to check with Sandeep if they want it for curated zone.